packages/playwright-core/types/structs.d.ts

```diff
@@ -19,7 +19,7 @@ import { JSHandle, ElementHandle, Frame, Page, BrowserContext } from './types';
 /**
  * Can be converted to JSON
  */
-export type Serializable = any;
+export type Serializable = unknown;
```

Comment:
Alternatively, we can make this more precise, but I'm not sure if there's really any benefit:

```ts
export type Json = boolean | number | string | null | JsonArray | JsonRecord
export interface JsonRecord {
  readonly [key: string]: Json
}
export interface JsonArray extends ReadonlyArray<Json> {}
/**
 * Can be converted to JSON
 */
export type Serializable = Json;
```

Reply:
@OliverJAsh I think `JsonRecord` could be typed as:

```ts
type JsonRecord = Record<string, Json>
```